src/baked-pasta-with-broccoli.md

```diff
@@ -0,0 +1,37 @@
+# Baked pasta with broccoli, boiled eggs and scamorza cheese
+
+![baked-pasta-with-broccoli](pix/baked-pasta-with-broccoli.webp)
+
+- ⏲️ Prep time: 15 min
+- 🍳 Cook time: 20~30 min
+- 🍽️ Servings: 4
+
+## Ingredients 
```

Comment:
```suggestion
## Ingredients
```

Trailing whitespace causes the script to fail. I have a PR to make explicitly check for this so that the error is actually clear, but it has been waiting for a merge for weeks.